docker/docker.md

```diff
@@ -0,0 +1,87 @@
+# sim-ln Docker Setup
+
+This guide provides instructions on how to build, run, and manage the `sim-ln` Docker container using the provided Makefile.
+
+## Prerequisites
+
+- Docker installed on your machine.
+- A `config.json` file that contains your specific configuration.
+
+## Building the Docker Image
+
+To build the Docker image, execute:
+
+```bash
+make build-docker
+```
+
+This command will make the necessary preparations and build the Docker image.
+
+## Mounting the Volume
+
+To ensure that the Docker container can access the configuration and other necessary data, we use Docker volumes. Before you can run the container, you need to set up the volume with the necessary data.
+
+To do this:
```

Comment:
I think that it would be helpful to note that we've provided an easy way to do this, but you can also do it yourself (eg in the case where the macaroons etc are on another machine). 

Something along the lines of: 
- If you're running nodes locally -> you can use this script, it'll copy everything and create a config suitable for the container. 
- If you're running nodes remotely, you'll need to provide your own volume and config that points to the files location *in the volume*. 

Can also add a note about IP addresses here.